x-pack/plugins/alerting/common/rrule_from_snooze.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RRule, ByWeekday, Weekday, rrulestr } from 'rrule';
```

Comment:
This file can't be in `/common` unless we want to increase our page bundle size limit. `rrule` is a big library so exposing it to the frontend breaks CI.

Until there's a very strong case for using `rrule` computations on the frontend I think we should move this to `/server`.